cmd/fluent-manager/main.go

```diff
@@ -98,10 +98,9 @@ func main() {
 	if watchNamespaces != "" {
 		namespacedController = true
 		namespaces := strings.Split(watchNamespaces, ",")
-		if len(namespaces) > 1 {
-			ctrlOpts.NewCache = cache.MultiNamespacedCacheBuilder(namespaces)
-		} else {
-			ctrlOpts.Namespace = namespaces[0]
+		ctrlOpts.Cache.DefaultNamespaces = make(map[string]cache.Config, len(namespaces))
+		for _, ns := range namespaces {
+			ctrlOpts.Cache.DefaultNamespaces[ns] = cache.Config{}
```

Comment:
https://github.com/kubernetes-sigs/controller-runtime/blob/main/designs/cache_options.md for reference.